src/components/Films/Films.tsx

```diff
@@ -24,16 +25,14 @@ export interface IItems {
   vote_average: number;
   vote_count: number;
 }
-
+interface Column {
+  title: string;
+  dataIndex: string;
+  key: string;
+}
```

Comment:
new line after, if it is an interface please prefix it with Upper case I lets sat IColumn as interface column